### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by odai-alali**
Hi @anvaka 

Thank you for this easy-to-use library. I am working on multilevel image zoom component and i found some things that need to be fixed for me.

Hope that this pull will be useful.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -428,9 +428,10 @@ function createPanZoom(domElement, options) {
     }
   }
 
-  function handleSignleFingerTouch(e) {
-    e.stopPropagation()
-    e.preventDefault()
+  function handleSingleFingerTouch(e) {
+    // allow single touch propagation to enable link or image maps touching
+    // e.stopPropagation()
```
Comment (anvaka) [inline index.js]: These lines  were here for a reason. We can't simply comment them out, as that would break consumers who don't want touch events be propagated, without giving them a way out.

I'd suggest to pass a callback into `options` object, and if it is present, invoke the callback here with `e` as an argument. Then it would be up to that callback to invoke `e.stopPropagation()`

Makes sense?

File: dist/panzoom.js [hunk 2]
```diff
@@ -175,19 +175,16 @@ function createPanZoom(domElement, options) {
     var boundingBox = getBoundingBox()
     if (!boundingBox) return
 
-    var adjusted = false
```
Comment (anvaka) [inline dist/panzoom.js]: Could you please elaborate why was this removed?

Comment (odai-alali) [reply dist/panzoom.js]: Hi anvaka
It is related to the change in these [two lines](https://github.com/anvaka/panzoom/pull/15/commits/5bcad6ef7d2f9f2ba6804205ebfcbbb92469bc84#diff-168726dbe96b3ce427e7fedce31bb0bcL286). the was disabled in case the transform was adjusted in the keepTransformInsideBounds() and this results in weird behavior in panzoom when the bounds set to true and the boundsPadding set to a value new or equal 1. I tried it with dom element and when the boundsPadding is 1, no zoom will be applied.

Removing this condition has no negative effect on the general behaviour of panzoom (I tested it on the dom element) and it allowed bounds to work properly and since the returned value is not needed when this condition is removed, I deleted the returned value in the function keepTransformInsideBounds()

File: index.js (hunk 1 above)
Comment (odai-alali) [reply index.js]: I needed to remove them to enable image maps areas touch. The callback solution makes sense for sure. I will try it out.

BTW, is there documentation for the library somewhere else than here? Yesterday I couldn't find documentation about options and bounds.

**Comment by odai-alali** [comment]
You can close this PR

I will send the commits in a separate PRs without building

**Comment by anvaka** [comment]
As discussed, closing it here.

File: index.js (hunk 1 above)
Comment (anvaka) [reply index.js]: The only documentation available is in the [README.md](https://github.com/anvaka/panzoom) file. Please feel free to extend it, and I'll do the same.
